ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -199,9 +201,11 @@ export const useNewMhrRegistration = () => {
 
   const fetchMhRegistrations = async (): Promise<void> => {
     const draftFilings = await getMhrDrafts()
-    const myMhrHistory = await mhrRegistrationHistory(true)
-    const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings)
-    setMhrTableHistory([...filteredMhrHistory])
+    const myMhrHistory = await mhrRegistrationHistory(isRoleQualifiedSupplier.value)
+    if (isRoleQualifiedSupplier.value === true) {
+      const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings)
+      setMhrTableHistory([...filteredMhrHistory])
```

Comment:
We can omit the spread operator because `filteredMhrHistory` is already an array.

Reply:
Made the change..